src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -138,17 +136,29 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("A space seperated list of functions to load.")
                 .Callback(f => EnabledFunctions = f);
 
-            return base.ParseArgs(args);
+            Parser
+                .Setup<bool>('v', "verbose")
+                .WithDescription("Sets Default LogLevel to Information.")
```

Comment:
I think something like "When false, hides system logs other than warnings and errors." Does that summarize it?